### PR TITLE
ConfigSet had missing logger

### PR DESCRIPTION
**Pull request by CGenie**

### CODE REVIEW

File: onedrive_d/od_glob.py [hunk 1]
```diff
@@ -118,6 +118,8 @@ class ConfigSet:
 		'LAST_RUN_TIMESTAMP': '1970-01-01T00:00:00+0000'
 	}
 
+	logger = get_logger()
+
```
Comment (xybu) [inline onedrive_d/od_glob.py:122]: Can change line 169 to `get_logger().info(...)` to make the code consistent.

**Comment by Dr-Xperience** [comment]
Well noob here so please be kind. The logging is not working I have an empty file in /var/log/onedrive_d.log. (Fedora 21)

**Comment by xybu** [comment]
The log is very likely to be written to stderr. Don't remember the old code base though.

Sincerely,
Xiangyu Bu

Sent from my Windows Phone

**Comment by Dr-Xperience** [comment]
The daemon doesn't write log on stderr. The instruction for old version i.e. this repository said to touch /var/log/onedrive_d.log file to maintain logs.

**Comment by CGenie** [comment]
Hm, I actually don't get the problems you're having. I checked out master, executed `onedrive-d start --debug`, all seems to work fine. I don't have the log in `/var/log/onedrive_d.log`, didn't create that file either and everything works, even though I have `"LOG_FILE_PATH": "/var/log/onedrive_d.log"` set in `config_v2.json`.

**Comment by xybu** [comment]
Honestly that param wasn't used by logger...

**Comment by CGenie** [comment]
Yeah, it seems that in the meanwhile code changed and there is no more `ConfigSet` class here. :)

**Comment by xybu** [comment]
I have forgotten almost everything in this repo...
Focused on this: https://github.com/xybu/onedrive-d

**Pull request merged**